packages/datasets-fs/src/flatten/flatten.ts

```diff
@@ -0,0 +1,76 @@
+import { Args } from './args'
+import { CaptchaItemTypes, Data, DataSchema, LabelledItem } from '@prosopo/types'
+import { ProsopoEnvError } from '@prosopo/common'
+import { blake2b } from '@noble/hashes/blake2b'
+import { consola } from 'consola'
```

Comment:
Switch to `import { logger } from @prosopo/common`

Reply:
done